label_studio/templates/base.html

```diff
@@ -126,10 +126,25 @@
 
 {% block bottomjs %}
   <script src="{{settings.HOSTNAME}}{% static 'js/jquery.min.js' %}"></script>
-  <script src="{{settings.HOSTNAME}}{% static 'js/helpers.js' %}"></script>
   <script src="{{settings.HOSTNAME}}{% static 'js/Toast.min.js' %}"></script>
   <script nonce="{{request.csp_nonce}}">
-      applyCsrf();
+
+      // CSRF
+      (function() {
+        function csrfSafeMethod(method) {
+          return (/^(GET|HEAD|OPTIONS|TRACE)$/.test(method));
```

Comment:
missed comment from original code, I'd better leave it here
```suggestion
          // these HTTP methods do not require CSRF protection
          return (/^(GET|HEAD|OPTIONS|TRACE)$/.test(method));
```